src/index.js

```diff
@@ -28,38 +29,74 @@ export function attach(env) {
   env.fetch = fetchWrapper(env.fetch);
 }
 
-function init() {
-  if (isReactNative()) {
-    attach(global);
-  } else if (isWorker()) {
-    attach(self);
-  } else if (isWeb()) {
-    attach(window);
-  } else if (isNode()) {
-    attach(global);
-  } else {
+function initialize() {
+  environment = resolveEnvironment();
+  if (!environment) {
     throw new Error('Unsupported environment for fetch-token-intercept');
   }
+
+  attach(environment);
 }
 
+/**
+ * Initializes and configures interceptor
+ * @param config
+ */
 export function configure(config) {
+  if (!interceptor) {
+    initialize();
+  }
+
   interceptor.configure(config);
 }
 
+/**
+ * Initializes tokens which will be used by interceptor
+ * @param args
+ */
 export function authorize(...args) {
   interceptor.authorize(...args);
 }
 
+/**
+ * Returns current set of tokens used by interceptor
+ * @returns {{accessToken: string, refreshToken: string}|*}
+ */
 export function getAuthorization() {
   return interceptor.getAuthorization();
 }
 
+/**
+ * Clears authorization tokens from interceptor
+ */
 export function clear() {
   return interceptor.clear();
 }
 
+/**
+ * Gets a value indicating whether interceptor is currently active
+ * @returns {boolean}
+ */
+export function isActive() {
+  return !!interceptor;
+}
+
+/**
+ * Removes interceptor and restores default behaviour
+ */
+export function unload() {
+  if (interceptor) {
+    interceptor.clear();
+    interceptor = null;
+  }
+
+  if (environment) {
+    environment.fetch = nativeFetch;
```

Comment:
This could be a bit dangerous, because you may overwrite the fetch configured by other libraries that were loaded after the `fetch-token-intercept` was configured. Maybe we should just stop intercepting the request, and leave the fetch as it is...